brigade/static/scss/pages/_brigade.scss

```diff
@@ -0,0 +1,4 @@
+// override style guide to give some more spacing here
+.list-icons li:not(:last-child) {
+  margin-bottom: 50px;
+}
```

Comment:
This is jank, I think on Monday I am going to redo this by converting the event list area to use an actual List atom class. (`.list--with-icon` or something like that).

Reply:
I agree, something like `.list--with-icons` makes sense to me.